automations/js/src/project_automation/issues.mjs

```diff
@@ -28,60 +34,75 @@ async function syncPriority(issue, board, card) {
  */
 export const main = async (octokit, context) => {
```

Comment:
We cannot directly import `@actions/core` into this file as we don't have direct access to Node.js modules. But `actions/github-script` provides `core` as a global that we can pass to `main` as an arg.

```suggestion
export const main = async (octokit, context, core) => {
```

`main` will then have to pass `core` to all functions just like `octokit` is being passed around.